### ISSUE TITLE
[CRD] monitor send string required

### ISSUE THREAD

**Issue by tkam8** [F5Networks/k8s-bigip-ctlr#1523]
#### Setup Details
CIS Version : 2.1.1
Build: f5networks/k8s-bigip-ctlr:latest
BIGIP Version: Big IP 14.1.0.1.0.0.7
AS3 Version: 3.22.1
Agent Mode: AS3
Orchestration: K8S
Orchestration Version: v1.18.6
Pool Mode: Nodeport

#### Description  
 
Documentation link below says send string is "Optional" but error when trying to deploy without send string:
https://clouddocs.f5.com/containers/latest/userguide/crd.html#health-monitor
 
Error: 
```
error: error validating "basic-single.yaml": error validating data: ValidationError(VirtualServer.spec.pools[0].monitor): missing required field "send" in com.f5.cis.v1.VirtualServer.spec.pools.monitor; if you choose to ignore these errors, turn validation off with --validate=false
```

Is it actually required or is documentation wrong? Please advise.

#### Steps To Reproduce
1)  Configure "vs" with monitor block but without send parameter
2) kubectl apply -f nameoffile.yaml
3) see stdout error

#### Expected Result 
Use default  send string

#### Actual Result  

Error

#### Diagnostic Information
```
apiVersion: "cis.f5.com/v1"
kind: VirtualServer
metadata:
  name: f5-nginx
  labels:
    f5cr: "true"
  namespace: default
spec:
  host: mysite.f5demo.com
  tlsProfileName: reencrypt-tls
  virtualServerAddress: "10.10.20.100"
  pools:
  - path: /bar
    service: nginx-svc1
    servicePort: 80
    monitor:
      type: http
      interval: 20
      timeout: 10
```

**Comment by iam-veeramalla** [comment]
Change already made and will be available in 2.2.

**Comment by mdditt2000** [comment]
Added to the list of documentation Sync Up for 2.2.0

**Comment by mdditt2000** [comment]
https://clouddocs.f5.com/containers/latest/userguide/crd.html#health-monitor -- Resolved in CIS 2.2
